flask_resty/api.py

```diff
@@ -16,6 +16,10 @@ def __init__(self, app=None, prefix=''):
             self._app = app
             self.init_app(app)
 
+        # store the view urls for reference. Doesn't support multiple routes
+        # mapped to same view
+        self.views = {}
```

Comment:
Uhm, I really need that additional information during `_add_resource`... I don't know what's the best place to store it though.... Creating another dictionary in `app.extensions`? :worried: